### ISSUE TITLE
Default sound heard during LiveRide

### ISSUE THREAD

**Issue by oliverlockwood** [cyclestreets/android#278]
Raised by @si-the-pie by email:

I tried Live Ride and instead of hearing a voice I got a default sound. The sound was emitted at the start of the ride and at each junction - so I'm wondering if I have to setup voice or some such. I have previously heard the voice on this device about a year ago.

Ah - now I'm home I have discovered that the media volume was low and the voice is actually there - but the default sound still comes too.  In fact there's a double default sound emitted at the start of the
route.

**Comment by oliverlockwood** [comment]
I believe this is down to the new notifications mechanism introduced since we migrated to API 26 (see https://github.com/cyclestreets/android/issues/214).  I'll dig into it.

**Comment by oliverlockwood** [comment]
Yup, I was right.  We were forced to set an importance level for the notifications, which naively I had set to "default".  From reading the documentation:
```
    /**
     * Low notification importance: shows everywhere, but is not intrusive.
     */
    public static final int IMPORTANCE_LOW = 2;

    /**
     * Default notification importance: shows everywhere, makes noise, but does not visually
     * intrude.
     */
    public static final int IMPORTANCE_DEFAULT = 3;
```

it becomes obvious this was the wrong setting.

PR to cover this shortly.

**Comment by oliverlockwood** [comment]
Reopening as @si-the-pie reports this as still not fixed.

Simon - what version of Android are you running?  Thanks.

**Comment by jezhiggins** [comment]
I'm not getting any sounds (and I turned my volume up just to be sure :+1: ), just the voice.  OnePlus 3 running Android 8.0.0.

**Comment by oliverlockwood** [comment]
I believe the discrepancy between @jezhiggins's experience and @si-the-pie's is perhaps due to https://stackoverflow.com/a/45920861/2108057.

This should be fixed in a PR I'll be raising shortly.

**Comment by oliverlockwood** [comment]
@si-the-pie can you please review whether the latest build fixes these issues, please?
